lms/djangoapps/verify_student/views.py

```diff
@@ -1280,13 +1280,15 @@ def results_callback(request):
 
     headers = {
         "Authorization": request.META.get("HTTP_AUTHORIZATION", ""),
+        "Content-Type": "application/json",
         "Date": request.META.get("HTTP_DATE", "")
     }
 
+    body_for_signature = {"EdX-ID": body_dict["EdX-ID"]}
```

Comment:
Why not keep it consistent and do `{"EdX-ID": body_dict.get("EdX-ID")}`? We can even avoid an exception this way if `EdX-ID` is not provided